schemas/common_definitions.json

```diff
@@ -301,5 +301,9 @@
   "non_empty_string": {
     "type": "string",
     "minLength": 1
+  },
+  "interviewer_note": {
+    "description": "Allows customisation of question and interstitial page title.",
+    "type": "boolean"
```

Comment:
The intention of the new property was to mark a content or question block as for the "interviewer only", rather than directly associate it with runners implementation. I think we should consider making the property `interviewer_only` and the description something like `Marks the page as for the interviewer only. Only applies for questionnaires that will be completed by an interviewer.`

Reply:
Done, thanks!